Public/Src/Engine/Scheduler/Scheduler.cs

```diff
@@ -1994,6 +2000,12 @@ public void UpdateStatus(bool overwriteable = true, int expectedCallbackFrequenc
                     writeFileDone: writeFileStats.DoneCount,
                     writeFileNotDone: writeFileStats.Total - writeFileStats.DoneCount - writeFileStats.IgnoredCount);
 
+                // Number of process pips that are not completed yet.
+                long numProcessPipsPending = m_processStateCountersSnapshot[PipState.Waiting] + m_processStateCountersSnapshot[PipState.Ready] + m_processStateCountersSnapshot[PipState.Running];
+
+                // PipState.Running does not mean that the pip is actually running. The pip might be waiting for a slot.
+                long numProcessPipsRunning = Workers.Sum(a => a.AcquiredSlotsForProcessPips);

```

Comment:
>Running [](start = 35, length = 7)

If PipState.Running doesn't mean "running" should we be reusing the ambiguous term running here?

I guess this feedback is bigger than the change in this PR but these states have become super confusing to the point where you have to hunt down the implementation that sets the value to know what it actually means. It'd be great to give some thought as to how to clean this up the future